### PR TITLE
add service for storing the `beforeinstallprompt`.

### PR DESCRIPTION
**Pull request by elsholz**

### CODE REVIEW

File: SAKPaaS/src/app/generated/api.module.ts [hunk 1]
```diff
@@ -16,7 +17,8 @@ import { OccupancyService } from './services/occupancy.service';
   providers: [
     LocationsService,
     OccupancyService,
-    ApiConfiguration
+    ApiConfiguration,
+    PwaRequestCatcherService
```
Comment (timgrohmann) [inline SAKPaaS/src/app/generated/api.module.ts:21]: Sollte ins `app.modle.ts`

File: SAKPaaS/src/app/app.component.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { TranslateService } from '@ngx-translate/core';
 import { CookieProviderService } from 'src/app/core/services/cookie-provider.service';
 import { MixpanelService, MixpanelId } from './core/services/mixpanel.service';
 import { DOCUMENT } from '@angular/common';
+import { PwaRequestCatcherService } from "./generated/services/pwa-request-catcher.service";
```
Comment (lepathie) [inline SAKPaaS/src/app/app.component.ts]: Useless import.

File: SAKPaaS/src/app/app.component.ts [hunk 3]
```diff
@@ -20,6 +21,7 @@ export class AppComponent implements OnInit {
   constructor(
     private locationService: LocationProviderService,
     private translate: TranslateService,
+    private pwaRequestCatcherService: PwaRequestCatcherService,
```
Comment (lepathie) [inline SAKPaaS/src/app/app.component.ts]: Useless parameter. Unused in class.

File: SAKPaaS/src/app/app.module.ts [hunk 4]
```diff
@@ -11,6 +11,8 @@ import { TranslateLoader, TranslateModule } from '@ngx-translate/core';
 import { TranslateHttpLoader } from '@ngx-translate/http-loader';
 import { loadConfig } from 'src/app/config-loader';
 import { ApiConfiguration } from 'src/app/generated/api-configuration';
+import { AddressPipe } from 'src/app/core/pipes/address.pipe';
```
Comment (timgrohmann) [inline SAKPaaS/src/app/app.module.ts:14]: :(

**Pull request merged**